notebooks/02-Summary-statistics-and-plots.ipynb

```diff
@@ -0,0 +1,512 @@
+    ")\n",
+    "props = dict(boxstyle=\"round\", alpha=0.5)\n",
+    "ax.text(\n",
+    "    0.5,\n",
+    "    0.85,\n",
+    "    text_str,\n",
+    "    transform=ax.transAxes,\n",
+    "    fontsize=12,  # verticalalignment=\"top\",\n",
+    "    horizontalalignment=\"center\",\n",
+    "    bbox=props,\n",
+    ")\n",
+    "# Return the index\n",
+    "df_three_images.set_index([\"image\", \"threshold\", \"molecule_number\"], inplace=True)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8f56b8fa-8c55-47a5-9d32-d461a1c639b6",
+   "metadata": {},
+   "source": [
+    "### Joint Plot\n",
+    "[Joint  Plots](https://seaborn.pydata.org/generated/seaborn.jointplot.html) showing the relationship between two variables can be plotted easily."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "6aa28c57-66fb-4857-b0df-9ac8f2725173",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "df.columns\n",
+    "sns.jointplot(data=df, x=\"min_feret\", y=\"max_feret\", hue=\"circular\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "300632da-8be7-4017-a958-f3ebbec01c1d",
+   "metadata": {},
+   "source": [
+    "# Loading Pickles\n",
+    "\n",
+    "One of the features of the `topostats.plotting` module is that it will save the graphs it has generated in Python's own\n",
+    "format for storing objects. These are known as [pickles](https://realpython.com/python-pickle-module/) and typically\n",
+    "have the file extension `.pkl`. After running TopoStats via `run_topostats` and plotting distributions of statistics a\n",
+    "`.pkl` will by default be saved in the output directory. Here we load (\"unpickle\") such a saved file which is typically\n",
+    "located at ~output/summary_distributions/distribution_plots.pkl~ and check its type (its a ~dict~ionary of plots).\n",
+    "\n",
+    "**NB** You will _have_ to have successfully processed some images with `run_topostats` to generate a pickle file and you\n",
+    "should adjust `Path()` in the cell below to reflect the location of this. You can use absolute or relative paths.\n",
+    "\n",
+    "TopoStats includes the ~load_pkl~ function which loads and returns the contents of the file."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "fded1124-b8f3-44b8-b73e-331079ebdce0",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pathlib import Path\n",
+    "\n",
+    "from topostats.io import load_pkl\n",
+    "\n",
+    "# NB - If you get an error saying the file could not be found then you need to modify this line to point to the location\n",
+    "# the file was saved to.\n",
+    "pkl_path = Path(\"../output/summary_distributions/distribution_plots.pkl\")\n",
+    "my_plots = load_pkl(pkl_path)\n",
+    "# Show the type of my_plots which is a dictionary of nested dictionaries\n",
+    "type(my_plots)"
```

Comment:
Something in this seems to print the pickle plots, which is fine, but they also don't have any axes or titles?

Reply:
:thinking: 

They do here...

![image](https://user-images.githubusercontent.com/17566406/223123320-959052cc-807b-47a4-9b91-ffbbdef26ce0.png)
![image](https://user-images.githubusercontent.com/17566406/223123349-908a40a5-95ed-4da9-8f04-1c74d6166e01.png)

Reply:
Yep just my scrolling issue but are saved as expected 👍